reference/upgrade/README.md

```diff
@@ -72,6 +72,8 @@ As we are aiming to always provide you more control and flexibility on how to de
 
 The `web` key is now a set of `locations` where you can define very precisely the behavior of each URL prefix.
 
+Note, we no longer move your application from "/" to "public/" automatically if the new format is adopted.
```

Comment:
You could
* Have it in `public` in the first place (in the repository)
* Possibly a good move command would be: `rsync --archive --exclude=public . public`